src/components/app/ProfileSharing.js

```diff
@@ -241,8 +223,8 @@ class ProfileSharingCompositeButton extends PureComponent {
         />
         <ButtonWithPanel
           className="profileSharingUploadErrorButton"
-          ref={this._uploadErrorButtonCreated}
           label="Upload Error"
+          open={!!error}
```

Comment:
I replaced one `openPanel` call with this `open` property. In the case of an error it's easy, but I didn't do it for other instances.